client/js/components/core/DpEditor/libs/handleWordPaste.js

```diff
@@ -273,10 +273,10 @@ function buildListAsHtmlString (list) {
  */
 function prepareDataBeforeParsingMso (slice) {
   return slice
-    // Strip line breaks
-    .replace(/(&nbsp;|\r|\n)/gmi, ' ')
     // Strip head
     .replace(/<head>(.|\n|\r)*?<\/head>/mi, '')
+    // Strip line breaks
+    .replace(/(\r\n)+?/gmi, ' ')
```

Comment:
Did you drop the `&nbsp;` intentionally? On first glance, I feel like we should keep that as well, just in case.

Reply:
no. It just doesn't makes any sense here. If Someone inserted a nonebreaking-space, it shall remain and will be handled by tiptap.